examples/plot_plz_areas.py

```diff
@@ -53,6 +53,7 @@
 
 ax.set_title('Inhabitants per Square Kilometer')
 
-fig.colorbar(plot, ax=ax)
+sm = plt.cm.ScalarMappable(norm=plt.Normalize(vmin=df['inhabitants_per_area'].min(), vmax=df['inhabitants_per_area'].max()))
```

Comment:
Can you comment why this was necessary? the PatchCollection returned by `plot_plz_data` should *just work*™ with colorbar.

Reply:
Ok, found out that this is a matplotlib bug triggered when a `pd.Series` is given as array for the `PatchCollection`.

Just adding `data = np.asanyarray(data)` in the `plz_plot` function solves this and makes this workaround unecessary.

I am going to cherry pick your other two commits, and fix this using the approach above.